app/views/transactions/new.js.erb

```diff
@@ -0,0 +1,16 @@
+$('#modal_title').html("<i class='glyphicon glyphicon-plus'></i> Add New Transaction");
```

Comment:
@yunixon I think code will be much simplier if you will render whole modal window in new/edit actions instead rendering each tab. So you will need only two lines: render modal content, show modal.